command/meta.go

```diff
@@ -77,12 +79,12 @@ func (m *ApiMeta) GlobalFlagSet(cmd string) *flag.FlagSet {
 
 	f.StringVar(&m.gateEndpoint, "gate-endpoint", "http://localhost:8084",
 		"Gate (API server) endpoint")
-
 	f.BoolVar(&m.ignoreCertErrors, "insecure", false, "Ignore Certificate Errors")
-
 	f.BoolVar(&m.quiet, "quiet", false, "Squelch non-essentual output")
-
 	f.BoolVar(&m.Color, "no-color", true, "Disable color")
+	// TODO(jacobkiefer): Codify the json-path as part of an OutputConfig or
+	// something similar. Sets the stage for yaml output, etc.
+	f.StringVar(&m.JsonPath, "jsonpath", "", "Filter a subpath of the output")
```

Comment:
I recommend making this a parameter to an `--output` or `-o` flag since we will likely want to support a number of output formats w/o cluttering the flag list.